doc/sphinx/addendum/extraction.rst

```diff
@@ -399,6 +399,97 @@ As an example of translation to a non-inductive datatype, let's turn
 
    Extract Inductive nat => int [ "0" "succ" ] "(fun fO fS n -> if n=0 then fO () else fS (n-1))".
 
+Generating FFI Code
+~~~~~~~~~~~~~~~~~~~
+
+The plugin provides mechanisms to generate (currently only) OCaml code to
+interface the generated OCaml code with C programs. In order to link compiled
+OCaml code with C code, the linker needs to know
+
+ * which C functions will be called by the ML code (external)
+ * which ML functions shall be accessible by the C code (callbacks)
+
+.. cmd:: Extract Foreign Constant @qualid => @string
+
+   Same as the :cmd:`Extract Constant`, except that the given ML terms
```

Comment:
```suggestion
   Like :cmd:`Extract Constant`, except that the referenced ML terms
```

"Same, ... except" is a bit contradictory.
The command has the qualid of the term, not the term itself.